app/components/work_show_info_component.rb

```diff
@@ -111,6 +115,12 @@ def related_works
       ).includes(:leaf_representative).all
   end
 
+  # This triggers a single request to SOLR to retrieve works deemed similar.
+  # This is unusual, but calling the code in the controller turned out to be unduly complex.
```

Comment:
can you put a comment at the TOP of this component .rb file too, warning that rendering it will, unusually, trigger a solr fetch?

Other than that, approved!